doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -835,27 +802,72 @@ Inductive definitions
 
 We gradually explain simple inductive types, simple annotated inductive
 types, simple parametric inductive types, mutually inductive types. We
-explain also co-inductive types.
+also explain co-inductive types.
 
 Simple inductive types
 ~~~~~~~~~~~~~~~~~~~~~~
 
-.. cmd:: Inductive @ident : {? @sort } := {? | } @ident : @type {* | @ident : @type }
+.. cmd:: Inductive @inductive_definition {* with @inductive_definition }
```

Comment:
I get your point but I'm not sure where to reinsert "We gradually..." and "Simple inductive types".  Seems like this may need some new text.

Reply:
> Seems like this may need some new text.

Agreed. Do put something that makes sense even if not perfect. I can try to help making it better, but overall the changes that this PR brings are worth degrading the text a little.